pulpcore/app/models/storage.py

```diff
@@ -164,3 +164,14 @@ def get_tls_path(model, name):
         type(model).__name__,
         str(uuid4()),
         name)
+
+
+try:
+    # django-storages/boto3 are optional packages
+    from storages.backends.s3boto3 import S3Boto3Storage
+
+    class S3Storage(S3Boto3Storage):
```

Comment:
Why don't we have users use `storages.backends.s3boto3.S3Boto3Storage` directly?

Reply:
We could but then we couldn't introduce custom logic or settings later on without having users change their configuration. If we're sure that we're not going to need to do that later on, then I am fine with making that change.

Reply:
I don't know if we would do this in the future, but when we have a reason to we can introduce it's usage as a configurable setting with that version. I think removing it would be good.